### ISSUE TITLE
Make it easy to contribute edits / content to this site

### ISSUE THREAD

**Issue by msurguy** [processing/processing-pi-website#18]
Currently, there is no link to this Github repository so the users don't know how to contribute content or report issues with the site's content. 

I propose the following two solutions: 
- Add a link to current page's MD file to the bottom of the right sidebar (table of contents) 
- Add a link to the footer

@gohai how do you feel about this?

**Comment by gohai** [comment]
Good idea. I think the bottom of the right sidebar makes it more visible. Something like "Issues? Let us know" or "Not working? File an issue" or ...?

BTW: you can pre-populate fields such as the label in links to GitHub's new-issue page like so https://github.com/processing/processing/issues/new?labels=arm&assignees=gohai
